pdks/FinFET14nm_Mock_PDK/fabric_Cap.py

```diff
@@ -40,31 +41,8 @@ def roundup( x, p):
         self.last_y1_track = roundup( (self.y_number-1)*c_m2_p, m2_p)
         self.last_x1_track = roundup( (self.x_number-1)*c_m1_p, m1_p)
 
-        if False: # probably don't need this (Better to do recoloring if necessary)
-            if (self.y_number-1) % 2 != self.last_y1_track % 2:
-                print( "Bump up last_y1_track for color compatibility")
-                self.last_y1_track += 1 # so the last color is compatible with the external view of the cell
-            if (self.x_number-1) % 2 != self.last_x1_track % 2:
-                print( "Bump up last_x1_track for color compatibility")
-                self.last_x1_track += 1 # so the last color is compatible with the external view of the cell
-
         print( "last_x1_track (m1Pitches_standards)", self.last_x1_track, "last_y1_track (m2Pitch_standards)", self.last_y1_track)
 
-        #gcd = math.gcd( self.m2Pitch_narrow, self.m2Pitch_standard)
-        #print( "GCD,LCM,(LCM in m2Pitch_narrowes),(LCM in m2Pitch_standards) of m2Pitch_narrow (minimum) and m2Pitch_standard (devices)", gcd, self.m2Pitch_narrow, self.m2Pitch_standard, (self.m2Pitch_narrow*self.m2Pitch_standard)//gcd, self.m2Pitch_standard//gcd, self.m2Pitch_narrow//gcd)
-
-
-        self.m1 = self.addGen( Wire( 'm1', 'M1', 'v',
-                                     clg=ColoredCenterLineGrid( colors=['c1','c2'], pitch=p['M1']['Pitch'], width=p['M1']['Width']),
-                                     spg=EnclosureGrid( pitch=p['M2']['Pitch'], stoppoint=p['V1']['VencA_L'] +p['M2']['Width']//2, check=True)))
-        self.m2 = self.addGen( Wire( 'm2', 'M2', 'h',
-                                     clg=ColoredCenterLineGrid( colors=['c1','c2'], pitch=p['M2']['Pitch'], width=p['M2']['Width']),
-                                     spg=EnclosureGrid( pitch=p['M1']['Pitch'], stoppoint=p['V1']['VencA_H'] + p['M1']['Width']//2, check=True)))
-        # cheating here width is the cap width; do not port m3
-        self.m3 = self.addGen( Wire( 'm3', 'M3', 'v',
-                                     clg=ColoredCenterLineGrid( colors=['c1','c2'], pitch=p['M3']['Pitch'], width=p['Cap']['m3Width']),
-                                     spg=EnclosureGrid( pitch=p['M2']['Pitch'], stoppoint=p['V2']['VencA_H'] +p['M2']['Width']//2, check=True)))
-
```

Comment:
Removing redundant code by using existing metal layers in DefaultCanvas.

@arvuce22  can you check to make sure the resulting Cap looks right?

Can you also try and identify any other code that can be merged / removed? I will add the m*n layers to the default canvas definition and add a method to generate cap arrays in primitive.py

Reply:
I will check the capacitor layouts